### PR TITLE
Tech Debt(Fix Redux state import/export) 🎉

### PR DESCRIPTION
**Pull request by schnogz**
## Description
- Fixes deserialization problems with app Redux State
- Removes no longer needed Lockbox invitation selectors and logic

## Change Type
- Tech Debt

## Testing Steps
- Pull branch and login to app and do a bunch of actions, export state, logout and then import state.

## Code Checklist
- [ ] Code compiles successfully (verified via `yarn start`)
- [ ] `README.md` and other documentation is updated as needed

### CODE REVIEW

**Comment by plondon** [comment]
Honestly, I know it saves time doing it the other way, but I prefer this because it makes redux state more explicit and easier to follow.

File: packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/reducers.js [hunk 1]
```diff
@@ -29,17 +29,49 @@ export default (state = INITIAL_STATE, action) => {
...
+      return assoc('states', Remote.Failure(payload.e), state)
+    }
+    case AT.SET_KYC_FLOW_LOADING: {
+      return assoc('flowConfig', Remote.Loading, state)
+    }
+    case AT.SET_KYC_FLOW_SUCCESS: {
+      return assoc('flowConfig', Remote.Success(payload.flowType), state)
+    }
+    case AT.SET_KYC_FLOW_FAILURE: {
+      return assoc('flowConfig', Remote.Failure(payload.e), state)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/reducers.js]: After reading the chapter on naming im tempted to say we should use err or error. Looks like we use error in most places

Comment (tony-blockchain) [reply packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/reducers.js]: I don't mind until we start shortening non-trivial stuff

Comment (tony-blockchain) [reply packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/reducers.js]: oh sorry, wrong comment, thought you'd like to make error => err
error is a slight win over failure 679 vs 443

**Pull request merged**